README.md

```diff
@@ -1,2 +1,3 @@
 # fiware-health
 Program to check the functionality of each FIWARE Lab node.
```

Comment:
Minor issue: should we use ReST instead of Markdown, as in the rest of components? Otherwise, we'll end up mixing the different markups.